src/SignalRServiceExtension/Config/StaticServiceManagerStore.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Azure.SignalR.Management;
+
+namespace Microsoft.Azure.WebJobs.Extensions.SignalRService
+{
+    /// <summary>
+    /// A global <see cref="IServiceManagerStore"/> for the extension.
+    /// It stores <see cref="IServiceManagerStore"/> per connection string.
+    /// </summary>
+    public static class StaticServiceManagerStore
```

Comment:
This should still be called StaticServiceHubContextStore

Reply:
Yes. Get() returns a ServiceHubContextStore